index.html

```diff
@@ -0,0 +1,155 @@
+
+            <h3>Jack</h3>
+            <h4>Cost to Adopt: $123.45</h4>
+            <p>Corrum volorit iandae nimaxim cum restia volor reicid ut et etur sunt arum rendae pla endis re ea erum, qui doluptae</p>
+            <a href="#link" class="Adopt-link" >Adopt</a>
+          </article>
+
+          <article>
+
+            <img class="dogimages" src="./images/duffy-card.jpg" alt="Black dog is staring at the camera" >
+
+            <h3>Duffy</h3>
+            <h4>Cost to Adopt: $123.45</h4>
+            <p>Corrum volorit iandae nimaxim cum restia volor reicid ut et etur sunt arum rendae pla endis re ea erum, qui doluptae</p>
+            <a href="#link" class="Adopt-link">Adopt</a>
+          </article>
+
+          <article class="div2">
+
+            <img class="dogimages" src="./images/lucas-card.jpg" alt="white dog is looking away" >
+
+            <h3>Lucas</h3>
+            <h4>Cost to Adopt: $123.45</h4>
+            <p>Corrum volorit iandae nimaxim cum restia volor reicid ut et etur sunt arum rendae pla endis re ea erum, qui doluptae</p>
+            <a href="#link" class="Adopt-link">Adopt</a>
+          </article>
+
+          <article class="div2">
+
+            <img class="dogimages" src="./images/jake-card.jpg" alt="pitbull is standing">
+
+            <h3>Jake</h3>
+            <h4>Cost to Adopt: $123.45</h4>
+            <p>Corrum volorit iandae nimaxim cum restia volor reicid ut et etur sunt arum rendae pla endis re ea erum, qui doluptae</p>
+            <a href="#link" class="Adopt-link">Adopt</a>
+          </article>
+
+          <article class="div2">
+
+            <img class="dogimages" src="./images/angus-card.jpg" alt="dog with black bakcground">
+
+            <h3>Angus</h3>
+            <h4>Cost to Adopt: $123.45</h4>
+            <p>Corrum volorit iandae nimaxim cum restia volor reicid ut et etur sunt arum rendae pla endis re ea erum, qui doluptae</p>
+            <a href="#link" class="Adopt-link">Adopt</a>
+          </article>
+// need to <!-- REVIEW:  -->
+          <article>
+
+            <img class="dogimages" src="./images/violet-card.jpg" alt="poppy brown dog sitting">
+
+            <h3>Violet</h3>
+            <h4>Cost to Adopt: $123.45</h4>
+            <p>Corrum volorit iandae nimaxim cum restia volor reicid ut et etur sunt arum rendae pla endis re ea erum, qui doluptae</p>
+            <a>Adopt</a>
+          </article>
+
+      </section>
+
+      <a>See More Dogs</a>
+
+    </main>
+
+
+    <footer>
+      <img class="footerlogo" src="./images/adoptadog-logo.svg" alt="footer-logo">
+      <p>Aborpos nisto inci tet aut que volupti beruptiberro to ex esseque invenienihil imaximu scideru ptistiusam consequunt </p>
+      <nav class="footer-nav-1">
+        <ul class="footer-nav-ul">
+          <li class="footer-nav-li">
+            <a href="#">Home</a>
+          </li>
+          <li class="footer-nav-li">
+            <a href="#">Dogs</a>
```

Comment:
You'll want to handle these / marks with a pseudo element in CSS. I'll invite you to investigate how to do that on your own.